exonum/src/storage/proof_list_index/mod.rs

```diff
@@ -24,13 +24,19 @@ use super::{
     indexes_metadata::IndexType,
     Fork, Snapshot, StorageKey, StorageValue,
 };
-use crypto::{hash, Hash, HashStream};
+use crypto::{Hash, HashStream};
+use byteorder::LittleEndian;
+use bytes::ByteOrder;
 
 mod key;
 mod proof;
 #[cfg(test)]
 mod tests;
 
+pub const LEAF_TAG: u8 = 0x0;
```

Comment:
Tags are intended to be global for all `Proof*Index`es, so it could make sense to move them upper in the module hierarchy.